scripts/setup.sh

```diff
@@ -182,6 +182,7 @@ if [ ! -d "$NODE_PATH" ]; then
   chown -R $ME $NODE_MODULE_DIR
   chmod -R 744 $NODE_MODULE_DIR
 fi
+$NPM_INSTALL
```

Comment:
Add a comment above this explaining what this installs (at least refer to package.json)